plugins/cd/__init__.py

```diff
@@ -151,7 +152,6 @@ def get_track_lengths(self):
 
 
 class CDPlaylist(playlist.Playlist):
-
```

Comment:
This type of change (remove empty line directly after the class definition) is not good imho. It is inconsistent with any other method. As far as I remember it also conflicts with the PEP8 rules.

Reply:
It appears that it only does this if the class doesn't have a docstring (which is generally recommended anyways).

For examples of the types of reformatting it does on various class bodies, see the tests at https://github.com/ambv/black/blob/master/tests/data/class_methods_new_line.py